Lib/multiprocessing/pool.py

```diff
@@ -271,11 +296,13 @@ def _maintain_pool(ctx, Process, processes, pool, inqueue, outqueue,
                        wrap_exception):
         """Clean up any exited workers and start replacements for them.
         """
-        if Pool._join_exited_workers(pool):
+        need_repopulate = Pool._join_exited_workers(pool)
+        if need_repopulate:
             Pool._repopulate_pool_static(ctx, Process, processes, pool,
                                          inqueue, outqueue, initializer,
                                          initargs, maxtasksperchild,
                                          wrap_exception)
+        return need_repopulate
```

Comment:
Mention the return value in the docstring?

Reply:
No need of returning this value after the rebase